docs/additional-configuration.adoc

```diff
@@ -351,3 +351,25 @@ spec:
 ----
 
 For documentation on Runtime Classes, see https://kubernetes.io/docs/concepts/containers/runtime-class/
+
+## Configuring the Webhook deployment
+The `devworkspace-webhook-server` deployment can be configured in the global DevWorkspaceOperatorConfig (DWOC). The configuration options include: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#replicas[replicas], https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/[pod tolerations] and https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector[nodeSelector]. 
+
+These configuration options exist in the **global** DWOC's `config.webhook`  field:
+
+```YAML
```

Comment:
This is using the wrong code block formatting style. It should be:
```adoc
[source,yaml]
----
(content)
----
```

Will fix this.